src/views/FindOrder.vue

```diff
@@ -235,19 +265,81 @@ export default {
     IonTitle,
     IonToolbar,
   },
+  computed: {
+    ...mapGetters({
+      orders: 'order/getOrders',
+      getProduct: 'product/getProduct',
+      currentFacilityId: 'user/getCurrentFacility',
+      getProductStock: 'stock/getProductStock',
+      isScrollable: 'order/isScrollable'
+    })
+  },
+  methods: {
+    async getOrders(vSize?: any, vIndex?: any){
+      const viewSize = vSize ? vSize : process.env.VUE_APP_VIEW_SIZE;
+      const viewIndex = vIndex ? vIndex : 0;
+      const payload = {
+        "json": {
+          "params": {
+            "rows": viewSize,
+            "start": viewSize * viewIndex,
+            "group": true,
+            "group.field": "orderId",
+            "group.limit": 10000,
+            "group.ngroups": true
+          } as any,
+          "query": "*:*",
+          "filter": "docType: ORDER AND orderTypeId: SALES_ORDER"
+        }
+      }
+      if (this.queryString) {
+        payload.json.params.defType = 'edismax'
+        payload.json.params.qf = 'orderId customerPartyName customerPartyId productId internalName'
+        payload.json.params['q.op'] = 'AND'
+        payload.json.query = `*${this.queryString}*`
+      }
+      await this.store.dispatch("order/findOrders", payload);
+    },
+    async copyToClipboard(text: any) {
+      await Clipboard.write({
+        string: text
+      }).then(() => {
+        showToast(this.$t('Copied', { text }));
+      })
+    },
+    async loadMoreOrders(event: any) {
+      this.getOrders(
+        undefined,
+        Math.ceil(this.orders.length / process.env.VUE_APP_VIEW_SIZE).toString()
+      ).then(() => {
+        event.target.complete();
+      })
+    }
+  },
+  mounted() {
+    this.getOrders();
+  },
   setup() {
     const router = useRouter();
+    const store = useStore();
+    const queryString = ref();
+    const orderStatus = reactive(JSON.parse(process.env.VUE_APP_ORDER_STATUS))
```

Comment:
I think we could skip making it reactive

Reply:
Sir, removed making the status object reactive.